db_upgrade_script/mosip_certify/sql/0.11.0_to_0.12.0_upgrade.sql

```diff
@@ -0,0 +1,253 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-- -------------------------------------------------------------------------------------------------
+-- Database Name: inji_certify
+-- Table Name : rendering_template,credential_template, ca_cert_store
+-- Purpose    : To upgrade Certify v0.11.0 changes and make it compatible with v0.12.0
+--
+-- Create By   	: Piyush Shukla
+-- Created Date	: January-2025
+--
+-- Modified Date        Modified By         Comments / Remarks
+-- ------------------------------------------------------------------------------------------
+-- ------------------------------------------------------------------------------------------
+
+-- Step 1: Rename the table
+ALTER TABLE certify.credential_template RENAME TO credential_config;
+
+-- Step 2: Add new columns
+ALTER TABLE certify.credential_config
+    ADD COLUMN credential_config_key_id VARCHAR(255) NOT NULL UNIQUE DEFAULT gen_random_uuid(),
```

Comment:
Critical: Can we have update statements to update this value based on appending the credential type and credential format ?